test/parallel/test-http-client-default-headers-exist.js

```diff
@@ -45,7 +45,7 @@ const server = http.createServer(common.mustCall((req, res) => {
          `${req.method} was an unexpected method`);
 
   const requestHeaders = Object.keys(req.headers);
-  requestHeaders.forEach((header) => {
+  requestHeaders.for((header) => {
```

Comment:
This won't work. You'll want to use a loop of the form `for (const header of requestHeaders) {` instead.